tutorials/developing-services-with-k8s.md

```diff
@@ -171,7 +171,7 @@ Terminate the PHP process, and type `exit` to terminate the Telepresence proxy a
 
 What's going on behind the scenes? Your incoming request goes to the load balancer. The load balancer, as mentioned above, looks for the Telepresence proxy based on the `app:guestbook` and `tier:frontend` labels. The proxy, which is running in the cloud Kubernetes environment, then sends those requests to the local Telepresence client, which passes the request to the PHP application.
```

Comment:
Replace cloud Kubernetes environment with Google Kubernetes Engine or GKE if you define it up above